dev/NavigationView/NavigationView.xaml

```diff
@@ -71,7 +71,13 @@
                             </VisualStateGroup>
 
                             <VisualStateGroup x:Name="TogglePaneGroup">
-                                <VisualState x:Name="TogglePaneButtonVisible" />
+                                <VisualState x:Name="TogglePaneButtonVisible">
+                                    <VisualState.Setters>
+                                        <!-- Might not need this setter when all the margins/paddings will be adequately set. -->
+                                        <!-- This MinHeight setter is only here to ensure that AutoSuggestBox doesn't draw over ToggelButton. -->
+                                        <Setter Target="PaneContentGridToggleButtonRow.MinHeight" Value="{StaticResource NavigationViewPaneHeaderRowMinHeight}"/>
+                                    </VisualState.Setters>
+                                </VisualState>
                                 <VisualState x:Name="TogglePaneButtonCollapsed" />
```

Comment:
nit: the default visual state should be the first one in the group

Reply:
Fixed now.